src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 PingCAP, Inc.
+// Copyright 2019 PingCAP, Inc.
```

Comment:
Why do you change this license?

Reply:
It's 2019 now

Reply:
I don't think it should be changed unless it is a new file.

Reply:
It's indeed a new fine 🤣introduced in 2019 but not modified in previous PRs.

Reply:
got it.